test/Datadog.Trace.ClrProfiler.IntegrationTests/CustomTestFramework.cs

```diff
@@ -22,6 +24,26 @@ public class CustomTestFramework : XunitTestFramework
         public CustomTestFramework(IMessageSink messageSink)
             : base(messageSink)
         {
+#if NETCOREAPP3_1
+
+            var paths = EnvironmentHelper.GetProfilerPathCandidates(null).ToArray();
+
+            foreach (var path in paths)
+            {
+                var baseDirectory = Path.GetDirectoryName(path);
+                var finalDirectory = Path.Combine(baseDirectory, "netcoreapp3.1");
+
+                if (Directory.Exists(finalDirectory))
+                {
+                    var file = typeof(Datadog.Trace.ClrProfiler.Instrumentation).Assembly.Location;
+                    File.Copy(file, Path.Combine(finalDirectory, Path.GetFileName(file)), true);
+                    return;
+                }
```

Comment:
I guess the alternative to the "hacky" replacement is to fix the assembly resolution to not require the duplicate assembly? easier said than done I guess

Reply:
Wouldn't we run into disk space issue if we stop sharing the assemblies?

Reply:
I meant the other way round, we're _already_ duplicating the lib (the reason you need to copy it). I meant we could just not do that duplication? Don't know if it's worth the effort though